### PR TITLE
Fix "object has no attribute 'tolist'"

### PR DESCRIPTION
**Pull request by jepler**
This occurs since the switch to pyopengl (yay!) but only if python3-numpy is not also installed.

I was able to eliminate the conversion of the matrix to its 1D / list form entirely.

I think I got pretty good manual testing coverage of everything _except_ qt vismach in my testing, both with and without python3-numpy installed.  qt vismach should be the same as tk vismach, as I made edits with the same intended effect.

### CODE REVIEW

File: lib/python/qtvcp/lib/qt_vismach/primitives.py [hunk 1]
```diff
@@ -654,8 +654,7 @@ def capture(self):
         # give a list of lists.
         # It would be better to change all the functions so a flat list could be used 
         # directly - i don't understand the code well enough
-        # self.t = GL.glGetDoublev(GL.GL_MODELVIEW_MATRIX)
-        self.t = list(itertools.chain.from_iterable(GL.glGetDoublev(GL.GL_MODELVIEW_MATRIX)))
+        self.t = glGetDoublev(GL_MODELVIEW_MATRIX))
```
Comment (rene-dev) [inline lib/python/qtvcp/lib/qt_vismach/primitives.py:657]: extra brace here causes the test to fail

**Comment by rene-dev** [comment]
looks good to me, thanks for looking into the issue

**Comment by jepler** [comment]
I'll correct the error soon, it looks trivial

**Comment by jepler** [comment]
I have a report on irc from @pcw-mesa that this didn't fix the problem, but I still think it's an improvement. (and don't see a problem here, running sim/axis/axis.ini and sim/axis/vismach/5axis/max5/max5kins.ini.  If you're still having a problem after this change, please report whether
 - installing python3-numpy makes a difference (I have no problem making this a hard dependency of linuxcnc)
 - the full traceback of where the error is still occurring
 - any steps you have to take before it occurs (my testing is just panning & rotating the view in axis, and running gcode to watch the backplot in vismach)

Feel free to report it here or in a fresh issue. If you mention me in a fresh issue I'll be more likely to see it.

**Pull request merged**